stable/keycloak/values.yaml

```diff
@@ -36,7 +36,9 @@ keycloak:
   ## Additional arguments to start command e.g. -Dkeycloak.import= to load a realm
   extraArgs: ""
 
-  ## Username for the initial Keycloak admin user
+  ## Config for the initial Keycloak admin user
+  admin:
+    create: false
```

Comment:
I think having this `true` is better since it keeps the existing functionality. Would probably be good to add a note to the README about how/why one is supposed to transition this from true to false.